cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -21,7 +21,7 @@
     <ul class="o-secondary-navigation_list
                o-secondary-navigation_list__parents">
         {%- for item in items %}
-            <li>
+            <li data-nav-is-active="{{ is_nav_active(request.url, item) }}">
```

Comment:
> It's just riskier because you are dependent upon the uniqueness of the slug.
 
I'm not understanding how uniqueness of the slug has anything to do with the parent's url being contained in the request url.

Also, this would be a more pythonic solution: `{{ request.url.startswith(item.url) }}`